### PR TITLE
SNMP/LLDP Containers: Sonic V2 Support

### PR DESCRIPTION
**Pull request by tombo315**
* Adding support for V2 in SNMP/LLDP (-sv2 postfix)
* Fixes for V1 containers: logging
* Fixes for V1 LLDP: limit LLDP to Front-panel or MGMT interfaces.

### CODE REVIEW

File: dockers/docker-lldp/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
-    dpkg_apt /deps/lldpsyncd_*.deb &&                                                           \
+    dpkg_apt /deps/lldpd_*.deb && \
+    dpkg_apt /deps/lldpsyncd_*.deb && \
     apt-get install -y python-pip supervisor && \
-    pip install /deps/*.whl && \
+    pip install /deps/sswsdk*.whl && \
     apt-get remove -y python-pip && \
     apt-get purge -y && apt-get autoclean -y && apt-get autoremove -y &&                        \
-    rm -rf /deps
+    rm -rf /deps ~/.cache
```
Comment (stcheng) [inline dockers/docker-lldp/Dockerfile]: align the '\'

File: dockers/docker-lldp/Dockerfile [hunk 2]
```diff
@@ -1,20 +1,22 @@
 FROM docker-base
 
-COPY deps/*py2*.whl deps/lldpsyncd_*.deb deps/lldpd_*.deb /deps/
+COPY deps/sswsdk*.whl deps/lldpsyncd_*.deb deps/lldpd_*.deb /deps/
+
+RUN apt-get update
```
Comment (qiluo-msft) [inline dockers/docker-lldp/Dockerfile]: >RUN [](start = 0, length = 3)

combine 2 RUN commands into one?

File: dockers/docker-lldp-sv2/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,23 @@
+FROM docker-base
+
+COPY deps/sswsdk*.whl deps/sonic_d*.whl deps/lldpsyncd_*.deb deps/lldpd_*.deb /deps/
+
+RUN apt-get update
```
Comment (qiluo-msft) [inline dockers/docker-lldp-sv2/Dockerfile]: >RUN [](start = 0, length = 3)

The same.

File: dockers/docker-snmp/Dockerfile [hunk 4]
```diff
@@ -6,6 +6,8 @@ COPY deps/python3/*.whl /python3/
 # enable -O for all Python calls
 ENV PYTHONOPTIMIZE 1
 
+RUN apt-get update
```
Comment (qiluo-msft) [inline dockers/docker-snmp/Dockerfile]: >RUN [](start = 0, length = 3)

The same.

File: dockers/docker-snmp-sv2/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    pip3 install --no-cache-dir /python3/*py3*.whl hiredis && \
+    rm -rf /python3 && \
+    python3 -m sonic_ax_impl install && \
+    python3 -m pip uninstall -y pip setuptools && \
+    rm -rf /usr/lib/python3.5/unittest && \
+    rm -rf /usr/lib/python3.5/lib2to3 && \
+    rm -rf /usr/lib/python3.5/tkinter && \
+    rm -rf /usr/lib/python3.5/idlelib && \
+    rm -rf /usr/lib/python3.5/email && \
+    rm -rf /usr/lib/python3.5/test && \
```
Comment (qiluo-msft) [inline dockers/docker-snmp-sv2/Dockerfile]: merge all 'rm' into one command: rm -rf /usr/lib/python3.5/{unittest,lib2to3,...}

**Comment by tombo315** [comment]
All comments addressed.

**Comment by stcheng** [comment]
please don't do merge. use squash merge instead.

**Pull request merged**